### PR TITLE
Make nonce action names unique and auto-generated

### PR DESCRIPTION
**Pull request by nabsul**
- The $action param is not the name of the hook, it should be as unique as possible to the request.
- We can make the action name unique based on a the name of the GET_VAR that it's protecting.
- This way, we don't need a different constant for each usage, we can generate an action name based on the GET_VAR.

Bonus change:
- Took all the constant strings out of functions and into const at the top of the class.

### CODE REVIEW

File: includes/class-wc-siftscience-admin.php [hunk 1]
```diff
@@ -20,9 +20,14 @@
...
-		private const ADMIN_LABEL = 'Sift';
-		private const NONCE_HOOK  = 'woocommerce_settings_siftsci';
+		private const ADMIN_ID            = 'siftsci';
+		private const ADMIN_LABEL         = 'Sift';
+		private const GET_VAR_CLEAR_STATS = 'clear_stats';
+		private const GET_VAR_SET_STATS   = 'set_siftsci_stats';
+		private const GET_VAR_RESET_GUID  = 'reset_guid';
+		private const GET_VAR_TEST_SSL    = 'test_ssl';
+		private const GET_VAR_CLEAR_LOGS  = 'clear_logs';
+		private const NONCE_ACTION_PREFIX = 'woocommerce_settings_nonce_';
```
Comment (ramico) [inline includes/class-wc-siftscience-admin.php:30]: do you imagine how long the HTML attrib would be

**Review by ramico — APPROVED** [review]
but try choosing a shorter prefix name

File: includes/class-wc-siftscience-admin.php (hunk 1 above)
Comment (nabsul) [reply includes/class-wc-siftscience-admin.php:30]: I think this is an internal string. It doesn't appear in an HTML attribute. But I can also make it shorted.

**Review by ramico — CHANGES_REQUESTED** [review]
vars must be only var name alone 
prefix should be wc_nonce_
removing word sift

**Pull request merged**